source/production/F0.Minesweeper.Components/Abstractions/MinefieldSize.cs

```diff
@@ -0,0 +1,6 @@
+namespace F0.Minesweeper.Components.Abstractions
+{
+	public sealed record MinefieldSize(uint Height, uint Width)
```

Comment:
I think it is more common that the `width` comes before the `height`.
Perhaps that would be "more natural", since also with our `Location`, we have `x` before `y`.

Reply:
That is very true. Exchanged height and width here and updated all found usages as well